#######################################################################
# This message was automatically generated by the lcm_to_ros package
# https://github.com/nrjl/lcm_to_ros, nicholas.lawrance@oregonstate.edu
#######################################################################
#
# Source message:    .msg
# Creation:          Thu 25 Aug 2022 10:31:21 AM EDT
#
#######################################################################
int64               utime   
float64[3]          ee_pose
float64[3]          ee_rpy
